public/components/integrations/components/__tests__/__snapshots__/setup_integration.test.tsx.snap

```diff
@@ -55,22 +36,8 @@ exports[`Integration Setup Page Renders integration setup page as expected 1`] =
                       }
                       integration={
                         Object {
-                          "assets": Object {
-                            "savedObjects": Object {
-                              "name": "sample",
-                              "version": "1.0.1",
-                            },
-                          },
-                          "components": Array [
-                            Object {
-                              "name": "logs",
-                              "version": "1.0.0",
-                            },
-                          ],
-                          "license": "Apache-2.0",
                           "name": "sample",
-                          "type": "logs",
-                          "version": "2.0.0",
+                          "type": "",
```

Comment:
Why are you removing type here

Reply:
Stopped filling a default value when the integration isn't loaded yet -- We only use the type when we go to integration adding so it's safe to keep empty until it's loaded.